packages/addons-inject/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@readthedocs/addons-inject",
+  "private": true,
+  "type": "module",
+  "version": "1.0.0",
+  "description": "Read the Docs Addons injection worker",
+  "main": "index.js",
+  "scripts": {
+    "dev": "wrangler dev --env dev",
+    "test": "vitest run --config vitest.config.js --reporter basic",
+    "lint": "prettier --check .",
+    "format": "prettier --write ."
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/readthedocs/common.git"
+  },
+  "author": "Read the Docs, Inc",
+  "license": "MIT",
+  "dependencies": {
+    "wrangler": "^3.82.0"
```

Comment:
Was there a reason behind using an old version?

Reply:
I don't remember exactly. I know that we needed _at least_ this version because some of the previous one had an issue that we were hitting.

I suppose it's not important to upgrade. We have tests now and it's working fine, so 👍🏼